crates/amaru/tests/conformance/stake-distributions/Makefile

```
@@ -48,15 +50,17 @@ unzst: check-network-dir ## Uncompress individual .json.zst files using zstd com
+check-env:
 	@test -n "$(ENDPOINT)" || { echo "Missing ENDPOINT"; exit 1; }
+
+check-aws-env: check-env
 	@test -n "$(AWS_ACCESS_KEY_ID)" || { echo "Missing AWS_ACCESS_KEY_ID"; exit 1; }
 	@test -n "$(AWS_SECRET_ACCESS_KEY)" || { echo "Missing AWS_SECRET_ACCESS_KEY"; exit 1; }
 
 list: check-network-dir check-aws-env ## List all objects in the network-specific S3 bucket
 	@aws s3 ls --recursive --human-readable --summarize --endpoint-url "$(ENDPOINT)" "s3://$(BUCKET)/"
 
-download: check-network-dir check-aws-env ## Download missing 'epoch_*.json.zst' fixtures from the network-specific S3 bucket
+download: check-network-dir check-env ## Download missing 'epoch_*.json.zst' fixtures from the network-specific S3 bucket
```

checkmake (0.3.2)

[warning] Target body for "download" exceeds allowed length of 5 lines (17).

(maxbodylength)